basyx.aasrepository-core/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/AasRepositorySuite.java

```diff
@@ -170,4 +174,32 @@ private Reference createDummyReference() {
 				.keys(new DefaultKey.Builder().type(KeyTypes.SUBMODEL).value(DUMMY_SUBMODEL_ID).build()).build();
 	}
 
+	@Test
+	public void getAssetInformation() {
+		assertEquals(aas2.getAssetInformation(), aasRepo.getAssetInformation(aas2.getId()));
+	}
+
+	@Test(expected = ElementDoesNotExistException.class)
+	public void getAssetInformationOfNonExistingAas() {
+		aasRepo.getAssetInformation("nonExisting");
+	}
+
+	@Test
+	public void setAssetInformation() {
+		AssetInformation assetInfo = createDummyAssetInformation();
+		aasRepo.setAssetInformation(aas2.getId(), assetInfo);
+		assertEquals(assetInfo, aasRepo.getAssetInformation(aas2.getId()));
+	}
+
+	private AssetInformation createDummyAssetInformation() {
```

Comment:
Please move the private function to the end of the file

Reply:
done